codegen-plugin/src/main/kotlin/io/spine/protodata/hello/BuilderValidationMethods.kt

```diff
@@ -0,0 +1,35 @@
+package io.spine.protodata.hello
+
+import com.google.common.collect.ArrayListMultimap
+
+/**
+ * A container to gather the validation methods that should be invoked
+ * in the `build` method of the message builder class.
+ */
+public class BuilderValidationMethods {
+
+    private val methodMap = ArrayListMultimap.create<String, String>()
+
+    /**
+     * Associates a builder validation method with the Java source file.
+     */
+    public fun linkMethod(javaSourceFile: String, validationMethod: String) {
```

Comment:
Please look through the Protodata's types (specifically, its Proto messages), maybe you could find something better than a `String` for `javaSourceFile`. It's a problem for every method in this class, but here (where there are two `String`-typed parameters) especially.

And also, on a separate subject, I would also document all parameters. It's really difficult to tell which values are expected.